File: 70.java
class Solution {
    public int climbStairs(int n) {
        int sum = 0;
        if (n == 1) {
            return 1;
        } else if (n == 2) {
            return 2;
        } else {
            int firstStair = 1;
            int secondStair = 2;
            
            for (int i = 2; i < n; i++) {
                sum = firstStair + secondStair;
                firstStair = secondStair;
                secondStair = sum;
            
            }
        }
        return sum;
    }
}

File: 70.py
class Solution:
    def climbStairs(self, n: int) -> int:
        if n == 1:
            return 1
        if n == 2:
            return 2
        firstStair = 1
        secondStair = 2
        sum = 0
        for i in range(2,n):
            sum = firstStair + secondStair
            firstStair = secondStair
            secondStair = sum
        return sum

s = Solution()
n = 4
print(s.climbStairs(n))
